client/me/help/help-courses/index.jsx

```diff
@@ -16,16 +16,52 @@ import {
 import { getCurrentUserId } from 'state/current-user/selectors';
 import { PLAN_BUSINESS } from 'lib/plans/constants';
 
+const courses = [
+	{
+		title: i18n.translate( 'How to Make a Business Site on WordPress.com' ),
+		description: i18n.translate(
+			'Bacon ipsum dolor amet jowl swine t-bone capicola porchetta andouille' +
+			'pork chop tail chicken. Ball tip frankfurter ham pastrami pork chop shank pork loin, bacon prosciutto. ' +
+			'Turducken pig meatloaf short ribs kielbasa. Chicken swine spare ribs ball tip pork loin cow drumstick ' +
+			'tail tongue venison tenderloin short ribs kielbasa salami. Andouille boudin prosciutto jowl tongue, bacon ' +
+			'chuck tail flank ball tip strip steak leberkas filet mignon. Tri-tip pancetta prosciutto spare ribs ball tip.'
+		),
+		schedule: [
+			{
+				date: i18n.moment( new Date( '8/25/2016 1:00 pm PST' ) ),
+				registrationUrl: 'http://wordpress.com'
+			},
+			{
+				date: i18n.moment( new Date( '8/27/2016 9:00 am PST' ) ),
+				registrationUrl: 'http://wordpress.com'
+			},
+			{
+				date: i18n.moment( new Date( '8/31/2016 10:30 am PST' ) ),
+				registrationUrl: 'http://wordpress.com'
+			},
+		],
+		recent: [
+			{
+				date: i18n.moment( new Date( '8/1/2016 10:30 am PST' ) ),
```

Comment:
Perhaps this should that be `PDT` for daylight savings time? I think if you use PST it'll force it to non-daylight savings, even if daylight savings would normally apply at that time (same with EST/EDT). Alternatively, using UTC should avoid this.

cc @dllh